tests/sympc/protocol/falcon/falcon_test.py

```diff
@@ -0,0 +1,6 @@
+from sympc.protocol import FALCON
+from sympc.tensor import ReplicatedSharedTensor
+
```

Comment:
Add a test for ensuring that Falcon protocol is recognized when passed in session. A test consisting of just 
`Session(protocol="FALCON")`

Reply:
Sure  @kamathhrishi